docs/team/sltsheryl.md

```diff
@@ -4,37 +4,63 @@ title: Sheryl-Lynn Tan's Project Portfolio Page
+Mass Linkers is a powerful Desktop application tool that provides a centralised platform for Computer Science (CS) students to find study support from batchmates with common modules. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java.
 
 Given below are my contributions to the project.
 
-* **New Feature**: [To be added soon]
-    * What it does: [To be added soon]
-    * Justification: [To be added soon]
-    * Highlights: [To be added soon]
-    * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+####New Enhancement: User Interface
+
+I was in charge of building the application's UI interface and handling the interaction between the ```model``` and ```ui``` components.
+1. Implemented a UI overhaul, with its new design inspired by Outlook. 
+2. Created module panel to display the modules taken by selected student. 
+3. Handled live updates to UI when for commands such as ```mod add```, ```mod mark``` and ```addInt```.
+4. Added color-coded tags to modules. 
+5. Redesigned interface of ```Person``` fields such as addition of GitHub and Telegram logos.
+
+**Changes to Files Made:**
+
+The following ```.fxml``` files were created or updated to reflect design changes.
+* ```PersonListCard``` to show a student information details
+* ```PersonListPanel``` to list students
+* ```MainWindow``` for the overall layout of the application and to listen for real-time updates on Person and Mod panels
+* ```ModListCard``` to show details of a module
+* ```ModListPanel``` to list modules
+
+Their respective ```.java``` files were also refactored to make the application more dynamic to changes. 
+
+The ```DarkTheme.css``` file was extensively refactored to update the UI interface.
+
+####New Feature: Add Interest
+
+I also worked on the add interest feature. The ```add int``` command gives users a hassle-free way of adding new interests to a student.
 
-* **New Feature**: [To be added soon]
+```AddInterestCommandTest``` and ```AddInterestParserCommandTest``` were also created to include testing coverage for ```AddInterestCommand``` and ```AddInterestParserCommand``` classes. 
 
-* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=sltsheryl&breakdown=true)
+####Other Modifications: Align Person with Application Purpose
+Modifications to the ```Person``` fields were made so it is tailored to the function of Mass Linkers, such as the removal of ```Address``` and addition of new fields like ```Telegram```and ```GitHub```. 
+####Code contributed
 
-* **Project management**:
-    * [To be added soon]
+[RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=sltsheryl&breakdown=true)
```

Comment:
This part didn't display correctly on my screen when I use the "Display  the rich diff" function at the top right corner under "Files changed" on GitHub. Not sure if the display is actually correct. Maybe you could double check the display for this part 🤔 :

####Other Modifications: Align Person with Application Purpose Modifications to the Person fields were made so it is tailored to the function of Mass Linkers, such as the removal of Address and addition of new fields like Telegramand GitHub. ####Code contributed

[RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=sltsheryl&breakdown=true)

Reply:
Okay I added space, that's the issue